android/src/test/java/com/instabug/instabugflutter/ArgsRegistryTest.java

```diff
@@ -156,6 +179,30 @@ private void assertAllInvocationOptionsArePresent(Map<String, Object> map) {
         Assert.assertTrue(map.containsValue(Option.EMAIL_FIELD_OPTIONAL));
     }
 
+    private void assertAllSupportedCustomTextPlaceHolderKeysArePresent(Map<String, Object> map) {
+        // source of truth
+        List<InstabugCustomTextPlaceHolder.Key> expectedKeys = getCurrentlySupportedKeysBySDK();
```

Comment:
We need to update our source of truth 🙏 
1. `VIDEO_PLAYER_TITLE`
2. `FEATURES_REQUEST`
3. `FEATURES_REQUEST_ADD_FEATURE_TOAST`
4. `FEATURES_REQUEST_ADD_FEATURE_THANKS_MESSAGE`
5. `SURVEYS_WELCOME_SCREEN_TITLE`
6. `SURVEYS_WELCOME_SCREEN_SUBTITLE`
7. `SURVEYS_WELCOME_SCREEN_BUTTON`
8. `REQUEST_FEATURE`